client_api/README.md

```diff
@@ -0,0 +1,19 @@
+# Gumbo Client API
```

Comment:
What do you think of calling this something like "Gumbo REST API"?  Specifically, having the word "client" in the name seems confusing on a first glance. (I understand it's a an api "for the client" but I think naming it something that communicates this is the service code would be a good idea)